core/src/ast_builder/expr/exists.rs

```diff
@@ -0,0 +1,56 @@
+use {super::ExprNode, crate::ast_builder::QueryNode};
+
+impl ExprNode {
+    pub fn exists<T: Into<QueryNode>>(self, query: T) -> Self {
+        exists(query)
+    }
```

Comment:
It does not use `self` at all.
So...
The proper location of `exists` might be not here.
May be it would be better for select nodes to have this. But it looks too big changes for now.
For this PR, I think it is simply enough without having `ExprNode::exists`

Reply:
Thanks good comment a lot I will try to fix it